docs/source/install/deb.rst

```diff
@@ -1,84 +1,210 @@
+.. contents::
+
+
+Minimal installation
+--------------------
+
+Install Dependencies
+~~~~~~~~~~~~~~~~~~~~
+
+Install MongoDB, RabbitMQ, and PostgreSQL.
+
+  .. code-block:: bash
+
+    sudo apt-get install -y mongodb-server rabbitmq-server postgresql
+
+
+Setup repositories
+~~~~~~~~~~~~~~~~~~~
+
+  .. code-block:: bash
+
+    wget -qO - https://bintray.com/user/downloadSubjectPublicKey?username=bintray | sudo apt-key add -
+    echo "deb https://dl.bintray.com/stackstorm/`lsb_release -cs`_staging stable main" | sudo tee /etc/apt/sources.list.d/st2-stable.list
+    sudo apt-get update
+
+
+Install StackStorm components
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  .. code-block:: bash
+
+      sudo apt-get update
+      sudo apt-get install st2 st2mistral
+
+
+If you are not running RabbitMQ, MongoDB or PostgreSQL on the same box, or changed defauls,
+please adjust the settings:
+
+    * RabbitMQ connection at ``/etc/st2/st2.conf`` and ``/etc/mistral/mistral.conf``
+    * MongoDB at ``/etc/st2/st2.conf``
+    * PostgreSQL at ``/etc/mistral/mistral.conf``
+
+Setup Mistral Database
+~~~~~~~~~~~~~~~~~~~~~~
+
+  .. code-block:: bash
+
+    # Create Mistral DB in PostgreSQL
+    cat << EHD | sudo -u postgres psql
+    CREATE ROLE mistral WITH CREATEDB LOGIN ENCRYPTED PASSWORD 'StackStorm';
+    CREATE DATABASE mistral OWNER mistral;
+    EHD
+
+    # Setup Mistral DB tables, etc.
+    /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head
+    # Register mistral actions
+    /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf populate
 
-.. warning::
+Configure SSH and SUDO
+~~~~~~~~~~~~~~~~~~~~~~
+To run local and remote shell actions, StackStorm uses a special system user (default ``stanley``).
+For remote linux actions, SSH is used. It is advised to configure identity file based SSH access on all remote hosts. We also recommend configuring SSH access to localhost for running examples and testing.
 
-   WOWZERS! A **deprecation** notice! Please note that this documentation and associated DEB packages are undergoing review and maintenance. Our core packages for Debian and Ubuntu are undergoing a complete overhaul to incorporate `dh-virtualenv <http://dh-virtualenv.readthedocs.org>`_ into our base packages. As such, please consider these instructions **DEPRECATED** for the time being. Hold tight, these packages and docs are coming very soon.
+* Take these steps on all boxes where you run stackstorm remote actions, **including** ``localhost``.
 
-   If you are feeling adventurous and want to attempt an installation using this method, we have left this documentation here for you to better understand the internals of the system as it currently exists
+  .. code-block:: bash
 
-   In the meantime, please feel free to take a look at many of our installation methods, including our `All-in-one Installer </install/all_in_one.rst>`_.
+    # Create an SSH system user
+    useradd stanley
```

Comment:
ack, decided to make it user / installer-script job. Use creating is fine (and when I follow the doc on the local box, it tells me that the user is created, and safely ignored, but note that exactly the same set applies to the remote boxes).